### ISSUE TITLE
clarity on multi class segmentation

### ISSUE THREAD

**Issue by DecentMakeover** [qubvel/segmentation_models.pytorch#118]
Hi

Thanks for sharing your work, 
for multi class is it enough to just add activation as softmax? or is there something more that has to be done?

Thanks in advance.
Ryan

**Comment by qubvel** [comment]
Should be fine with adding softmax, correct sampling and loss function.

**Comment by DecentMakeover** [comment]
Okay thanks
